cmd/semantic-release/main.go

```diff
@@ -47,6 +47,7 @@ func main() {
 	slug := flag.String("slug", os.Getenv("TRAVIS_REPO_SLUG"), "slug of the repository")
 	ghr := flag.Bool("ghr", false, "create a .ghr file with the parameters for ghr")
 	noci := flag.Bool("noci", false, "run semantic-release locally")
+	nochange := flag.Bool("nochange", false, "allow semver to run where the current commit is already a tagged release")
```

Comment:
I would clarify what 'allow' means - something like 'don't return an error code when the calculated version has already been tagged' might be a bit clearer

Reply:
Thanks I struggled a bit here on how to phrase this